presto-raptor/src/test/java/com/facebook/presto/raptor/metadata/TestDatabaseShardManager.java

```diff
@@ -147,6 +147,10 @@ public void testGetNodeTableShards()
         }
 
         shardManager.createTable(tableId, columns);
+        for (String node : nodes) {
```

Comment:
Remove this. It was testing that shards were dropped properly. No point in testing that it is empty after creation (since they couldn't possibly have been created)